extension/popup/ui.js

```diff
@@ -178,14 +187,18 @@ this.ui = (function() {
     settingsIcon.addEventListener("click", showSettings);
   }
 
-  function closePopup() {
+  exports.closePopup = function closePopup(ms = 2500) {
```

Comment:
What's the reason for the timeout here? Enough time for someone to read the result before closing? The time should at least get a name and comment at the top of the file.

Reply:
Yes, just to give people a bit of time to glance at it before it disappears. I'll make it a const